docs/community/com-cluster/com-cluster-change-partition-node.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Safely change parameters
+layout: default
+parent: Community cluster
+grand_parent: Community
+---
+
+# How do I safely change cluster parameters?
```

Comment:
Where did the emphasis on changing cluster parameters come from? I can't find this in the old cluster doc, so I'm trying to understand

Reply:
It's in com-config-flags -- https://docs.featurebase.com/docs/community/com-config/com-config-flags/#cluster-partition-to-node-assignment

Reply:
ah ok cool!